client/src/stores/audio-player.store.ts

```diff
@@ -58,10 +59,7 @@ export class AudioPlayerStore {
   handleAudioError() {
     this.hasError = true;
     this.hasLoaded = true;
-    this.rootStore.errorHandler.handle(
-      'Something went wrong while loading audio!',
-      'warning'
-    );
+    this.rootStore.errorHandler.handle(Errors.AudioLoadError, 'warning');
```

Comment:
Is it possible to move error types for errorHandler.handle (warning, error) to a separate enum?

Same as client/src/utils/httpClient.ts Line 98

Reply:
'warning' or 'error' are coming from the notification library that I'm using (`TypeOptions` type), and this type is a union of string literal types, so it gives me autocompletion. So, I don't think we should create a separate enum for that
https://github.com/Bogdan-Zinovij/Ecstasify/blob/69a0a77a856f9228d11e87e5912835712f7f6894/client/src/utils/errorHandler.ts#L4